general-knowledge/ethereum-blockchain/more-help-support-and-communities.md

```diff
@@ -17,40 +17,37 @@ date_modified: '2020-10-08'
 - Getting Started: [Reddit Guide](https://www.reddit.com/r/ethereum/comments/61y5ix/welcome_to_rethereum_the_reddit_front_page_of_the/)
 - for Noobs: [/r/ethereumnoobies](https://www.reddit.com/r/ethereumnoobies)
 - for Development: [/r/ethdev](https://www.reddit.com/r/ethdev)
-- Giveth Slack: [Giveth Slack](https://giveth.slack.com/)
 
 ## Clients / Wallets / Interfaces
 
-- Geth: [Gitter](https://gitter.im/ethereum/go-ethereum)
-- Mist: [Gitter](https://gitter.im/ethereum/mist)
-- Parity: [GitHub](https://github.com/paritytech/parity) · [Gitter](https://gitter.im/paritytech/parity)
-- Metamask: [Slack](https://metamask-slack-autoinvite.herokuapp.com/) · [Support](https://support.metamask.io/new)
-- MyCrypto: [Website](https://mycrypto.com/) · [FAQ](https://support.mycrypto.com/) · [Support](mailto:support@mycrypto.com)
+- Geth: [Website](https://geth.ethereum.org/) · [Gitter](https://gitter.im/ethereum/go-ethereum)
+- OpenEthereum: [GitHub](https://github.com/openethereum/openethereum)
+- Metamask: [Website](https://metamask.io/) · [Support](https://metamask.zendesk.com/hc/en-us)
+- MyCrypto: [Website](https://mycrypto.com/) · [FAQ](https://support.mycrypto.com/) · [Support](/contact-us)
 - MyEtherWallet: [Website](https://myetherwallet.com/) · [FAQ](https://myetherwallet.github.io/knowledge-base/) · [Support](mailto:support@myetherwallet.com)
-- Jaxx: [Website](https://jaxx.io/) · [Support](https://decentral.zendesk.com/hc/en-us)
-- Ledger Nano S: [Website](https://www.ledgerwallet.com/) · [Support](https://ledger.zendesk.com/hc/en-us/requests/new)
-- Trezor: [Website](https://trezor.io/) · [Gitter](https://gitter.im/trezor/community) · [Support](mailto:support@trezor.io)
+- Jaxx: [Website](https://jaxx.io/) · [Support](https://support.decentral.ca/hc/en-us)
+- Ledger: [Website](https://www.ledger.com/) · [Support](https://support.ledger.com/hc/en-us/requests/new)
```

Comment:
https://www.ledgerwallet.com/r/1985?path=/products/

Just change the main link here to include the referral, but leave the support page link as-is.